backend/src/API/candidateDeciderAPI.ts

```diff
@@ -89,6 +124,17 @@ export const getCandidateDeciderInstance = async (
   return instance;
 };
 
+/**
+ * Updates the rating and comment for a Candidate Decider instance.
+ * @param {IdolMember} user - The user updating the rating and comment.
+ * @param {string} uuid - The unique identifier of the instance.
+ * @param {number} id - The identifier of the specific candidate.
+ * @param {Rating} rating - The new rating.
+ * @param {string} comment - The new comment.
+ * @returns {Promise<void>} - A promise that resolves to void.
```

Comment:
I don't think you put this in the docstrings for the other methods in this file that have `Promise<void>` return type. Maybe remove it?